pkg/sdk/procedures_def.go

```diff
@@ -319,6 +325,211 @@ var ProceduresDef = g.NewInterface(
+	"https://docs.snowflake.com/en/sql-reference/sql/call",
+	g.NewQueryStruct("Call").
+		SQL("CALL").
+		Name().
+		PredefinedQueryStructField("CallArguments", "[]string", g.KeywordOptions().MustParentheses()).
+		PredefinedQueryStructField("ScriptingVariable", "*string", g.ParameterOptions().NoEquals().NoQuotes().SQL("INTO")).
 		WithValidation(g.ValidIdentifier, "name"),
+).CustomOperation(
+	"CreateAndCallForJava",
+	"https://docs.snowflake.com/en/sql-reference/sql/call-with#java-and-scala",
+	g.NewQueryStruct("CreateAndCallForJava").
+		SQL("WITH").
+		Identifier("Name", g.KindOfT[AccountObjectIdentifier](), g.IdentifierOptions().Required()).
+		SQL("AS PROCEDURE").
+		ListQueryStructField(
+			"Arguments",
+			procedureArgument,
+			g.ListOptions().MustParentheses(),
+		).
+		QueryStructField(
+			"Returns",
+			procedureReturns,
+			g.KeywordOptions().SQL("RETURNS").Required(),
+		).
+		SQL("LANGUAGE JAVA").
+		TextAssignment("RUNTIME_VERSION", g.ParameterOptions().SingleQuotes().Required()).
+		ListQueryStructField(
+			"Packages",
+			procedurePackage,
+			g.ParameterOptions().MustParentheses().SQL("PACKAGES").Required(),
+		).
+		ListQueryStructField(
+			"Imports",
+			procedureImport,
+			g.ParameterOptions().Parentheses().SQL("IMPORTS"),
+		).
+		TextAssignment("HANDLER", g.ParameterOptions().SingleQuotes().Required()).
+		PredefinedQueryStructField("NullInputBehavior", "*NullInputBehavior", g.KeywordOptions()).
+		PredefinedQueryStructField("ProcedureDefinition", "*string", g.ParameterOptions().NoEquals().SingleQuotes().SQL("AS")).
+		OptionalQueryStructField(
+			"WithClause",
+			procedureWithClause,
+			g.KeywordOptions(),
+		).
+		SQL("CALL").
+		Identifier("ProcedureName", g.KindOfT[AccountObjectIdentifier](), g.IdentifierOptions().Required()).
+		PredefinedQueryStructField("CallArguments", "[]string", g.KeywordOptions().MustParentheses()).
+		PredefinedQueryStructField("ScriptingVariable", "*string", g.ParameterOptions().NoEquals().NoQuotes().SQL("INTO")).
+		WithValidation(g.ValidateValueSet, "RuntimeVersion").
+		WithValidation(g.ValidateValueSet, "Handler").
+		WithValidation(g.ValidIdentifier, "ProcedureName").
+		WithValidation(g.ValidIdentifier, "Name"),
+).CustomOperation(
+	"CreateAndCallForScala",
+	"https://docs.snowflake.com/en/sql-reference/sql/call-with#java-and-scala",
+	g.NewQueryStruct("CreateAndCallForScala").
+		SQL("WITH").
+		Identifier("Name", g.KindOfT[AccountObjectIdentifier](), g.IdentifierOptions().Required()).
+		SQL("AS PROCEDURE").
+		ListQueryStructField(
+			"Arguments",
+			procedureArgument,
+			g.ListOptions().MustParentheses(),
+		).
+		QueryStructField(
+			"Returns",
+			procedureReturns,
+			g.KeywordOptions().SQL("RETURNS").Required(),
+		).
+		SQL("LANGUAGE SCALA").
+		TextAssignment("RUNTIME_VERSION", g.ParameterOptions().SingleQuotes().Required()).
+		ListQueryStructField(
+			"Packages",
+			procedurePackage,
+			g.ParameterOptions().MustParentheses().SQL("PACKAGES").Required(),
```

Comment:
The reviewdog seems to be right, there is no such method on ParameterTransformer

Reply:
needed to update branch, this was added in a previous PR